### PR TITLE
lint

### PR DESCRIPTION
**Pull request by kghamilton89**
Going through Python repo for blacklisted words and phrases. I imagine the linter will pick up a lot more once it runs through after I open the PR.

### CODE REVIEW

**Comment by kghamilton89** [comment]
* `./` prefix added for relative links.
* "prettytable" added to dictionary.

**Comment by kghamilton89** [comment]
Travis reads links to specific lines in a `.py` (and probably other code) documents such as `./atsd_client/services.py#52` as an error for `no such heading`, I'll go through and replace the headings in the repo so that this error is resolved.

File: offline_installation.md [hunk 1]
```diff
@@ -80,7 +80,7 @@ Copy the `atsd-api-python` and `modules` directories from the intermediate serve
 
 ## Install Modules on the Target Server
 
-Login into the target server where the ATSD client will be installed.
+Log in into the target server where the ATSD client is being installed.
```
Comment (raipc) [inline offline_installation.md]: `Log in to` or `log in into`? Maybe we can create a rule for that?

Comment (raipc) [reply offline_installation.md]: I think we should be able to link to particular line in source code. I will try to fix the linter to allow it.

File: atsd_client/models/_data_queries.py [hunk 2]
```diff
@@ -332,14 +334,14 @@ def __init__(self, limit=None, direction=None, series_limit=None, cache=None, re
         self.direction = "DESC" if direction is None else direction
         #: `int` maximum number of series returned. Default: 0.
         self.seriesLimit = 0 if series_limit is None else series_limit
-        # : `bool` flag. If true, execute the query against Last Insert table which results in faster response time
+        # : `bool` option. If true, execute the query against Last Insert table which results in faster response time
```
Comment (raipc) [inline atsd_client/models/_data_queries.py]: Seems weird to me.
https://techterms.com/definition/flag
https://trends.google.com/trends/explore?date=today%205-y&q=boolean%20flag,boolean%20option,boolean%20parameter

File: examples/find_lagged_series.py [hunk 3]
```diff
@@ -30,8 +30,8 @@
         series_list = metrics_service.series(m.name, entity.name)
         # for each list with more than 1 series
         if len(series_list) > 1:
-            # calculate maximum of all last_insert_date's in list and subtract 1 hour
-            # it will be lower limit date to compare
+            # calculate maximum of all last_insert_date samples in list and subtract 1 hour
```
Comment (raipc) [inline examples/find_lagged_series.py]: The meaning has changed. Was like the following.
`Set lower limit to maximum last insert date over series list minus 1 hour`

File: examples/find_non-positive_values.py [hunk 4]
```diff
@@ -30,7 +30,7 @@
 series = svc.query(query)
 
 if deleteValues:
-    print('Inappropriate values will be deleted.\n')
+    print('Inappropriate values deleted.\n')
 else:
     print('Leave as is inappropriate values.\n')
```
Comment (raipc) [inline examples/find_non-positive_values.py]: inversion

File: examples/frequency_violate.py [hunk 5]
```diff
@@ -53,7 +53,7 @@ def resolve_frequency(frequency):
 print('value, actual date, expected date, frequency, metric, entity, tags')
 for metric in metric_list:
 
-    # get metric frequency and trnasform it into relativedelta
+    # get metric frequency and trnasform frequency into relativedelta
```
Comment (raipc) [inline examples/frequency_violate.py]: transform

File: examples/migration.py [hunk 6]
```diff
@@ -8,7 +8,7 @@
 from atsd_client.services import SeriesService
 
 '''
-The test should compare series query responses for the given metric, entity, tags, and start/end dates before and
+The test compares series query responses for the given metric, entity, tags, and start/end dates before and
 after ATSD migration, compare results.
```
Comment (raipc) [inline examples/migration.py]: pleonasm?

File: offline_installation.md (hunk 1 above)
Comment (kghamilton89) [reply offline_installation.md]: It doesn't matter now, all links have already been replaced.

Comment (kghamilton89) [reply offline_installation.md]: Changed.

File: atsd_client/models/_data_queries.py (hunk 2 above)
Comment (kghamilton89) [reply atsd_client/models/_data_queries.py]: flag is a blacklisted word

File: atsd_client/models/_data_models.py [hunk 7]
```diff
@@ -142,7 +142,7 @@ def __init__(self, entity, metric, data=None, tags=None, last_insert_date=None,
                     )
                 else:
                     self._data.append(data_unit)
-        # : `datetime` object | `long` milliseconds | `str` ISO 8601 date. Last time a value was received for this
+        # : `datetime` object | `long` milliseconds | `str` ISO 8601 date. Last time a value is received for this
```
[the remark this answers is not in the document]
Comment (kghamilton89) [reply atsd_client/models/_data_models.py]: Changed.

File: examples/find_lagged_series.py (hunk 3 above)
Comment (kghamilton89) [reply examples/find_lagged_series.py]: changed.

File: examples/find_non-positive_values.py (hunk 4 above)
Comment (kghamilton89) [reply examples/find_non-positive_values.py]: changed.

File: examples/frequency_violate.py (hunk 5 above)
Comment (kghamilton89) [reply examples/frequency_violate.py]: :+1:

File: examples/migration.py (hunk 6 above)
Comment (kghamilton89) [reply examples/migration.py]: Good point, shortened.

**Comment by raipc** [comment]
Please, bring back the links to particular line of code. Current anchors don't work.

**Comment by kghamilton89** [comment]
@raipc link changes reverted.

**Pull request merged**